### PR TITLE
Added Secrets to Helm Chart

### PR DESCRIPTION
**Pull request by KevinMellott91**
Integrated Kubernetes secrets into the Helm chart to secure the Postgres database credentials.

### CODE REVIEW

File: chart/templates/marquez/deployment.yaml [hunk 1]
```diff
@@ -59,7 +59,10 @@ spec:
             - name: POSTGRES_USER
               value: {{ .Values.marquez.db.user }}
             - name: POSTGRES_PASSWORD
-              value: {{ .Values.marquez.db.password }}
+              valueFrom:
+                secretKeyRef:
+                  name: {{ include "marquez.secretName" . }}
+                  key: marquez-password
```
Comment (wslulciuc) [inline chart/templates/marquez/deployment.yaml:65]: Mind if we use the name `marquez-db-password` to make it clear it's for the database?

File: chart/values.yaml [hunk 2]
```diff
@@ -18,6 +18,10 @@ marquez:
     repository: marquezproject/marquez
     tag: 0.12.2
     pullPolicy: IfNotPresent
+  ## Name of the existing secret containing credentials for the Marquez installation.
+  ## When this is specified, it will take precedence over the values configured in the 'db' section.
+  ##
+  existingSecret: marquez-secrets
```
Comment (wslulciuc) [inline chart/values.yaml:24]: Mind if we use `existingSecretName` to be more aligned with the naming `marquez.secretName`?

**Comment by wslulciuc** [comment]
Really amazing work, @KevinMellott91! Had a couple of minor naming comments, but looks good to merge once addressed :)

**Comment by KevinMellott91** [comment]
@wslulciuc I feel like a git command wizard now, but this PR should be cleaned up and good to go :)

**Review by wslulciuc — APPROVED** [review]
LGTM 💯 💯 💯

**Pull request merged**